MainPage.xaml

```diff
@@ -122,10 +125,10 @@
                                     Margin="5,0,0,3"
                                     Grid.Column="1"
                                     VerticalAlignment="Bottom"
-                                    Text="{x:Bind FullName}" />
+                                    Text="{Binding FullName}" />
```

Comment:
@kimchhunnn Why did you change from x:Bind to Binding ?

Reply:
@kimchhunnn When I moved Repo class out of MainPage.xaml.cs, x:bind throws an error when run

Reply:
@kimchhunnn Do you know the different between both ?

Reply:
@kenphanith Yes, I used Binding because I don't have to create an object and mention the path to the type I want to use.